lib/developer_portal/app/views/developer_portal/invoices/show.html.liquid

```diff
@@ -18,7 +18,9 @@ table#line-items {
         {{ invoice.period_begin | date: '%B, %Y' }} Invoice
       </div>
       <div class="panel-body panel-footer">
-
+        {% if invoice.payment_transactions.size > 0 %}
```

Comment:
This should probably change to something that checks that state of the invoice and if a client secret can later be obtained from an existing pending payment intent, which actually would require an new method in https://github.com/3scale/porta/blob/485125888a3208188da19062c289ffabe55ebcca/lib/developer_portal/lib/liquid/drops/invoice.rb#L157

E.g.:

```ruby
module Liquid
  module Drops
    class Invoice < Drops::Model
      # ...
      def pay_now?
        @invoice.pending? && @invoice.payment_intents.pending.any?
      end
      # ...
    end
  end
end
```

```suggestion
        {% if invoice.pay_now? %}
```